actions.py

```diff
@@ -100,10 +103,18 @@ def push_notification(
             )
         else:
             notification.save()
+
+            if send_only_to_subscribed_targets:
+                persons = UserSubscription.objects.filter(
+                    category=category
+                ).filter(person__id__in=persons)\
```

Comment:
Not sure of this, but would default persons=None throw an error here? Might be better to use an empty array

Reply:
Actually, we should fetch it from redisdb like this: https://github.com/IMGIITRoorkee/omniport-service-emails/blob/3a98f640d4578372519cc4f34ea3f1fd2ab112cd/actions.py#L74

Reply:
And then we can take an intersection of both.